utf8::decode() decodes its argument in place and returns a boolean.
https://github.com/formbuilder/formbuilder/pull/15

Index: lib/CGI/FormBuilder/Source/File.pm
--- lib/CGI/FormBuilder/Source/File.pm.orig
+++ lib/CGI/FormBuilder/Source/File.pm
@@ -79,7 +79,7 @@ sub parse {
         chomp;
         my($term, $line) = split /\s*:\s*/, $_, 2;
         $utf8 = 1 if $term eq 'charset' && $line =~ /^utf/;  # key off charset to decode value
-        $line = utf8::decode($line) if $utf8;
+        utf8::decode($line) if $utf8;
 
         # here string term-inator (har)
         if ($here) {
